lib/gui/app/pages/main/MainPage.tsx

```diff
@@ -163,36 +163,26 @@ export class MainPage extends React.Component<
 		return (
 			<>
 				<Flex
-					id="app-header"
-					justifyContent="center"
-					style={{
-						width: '100%',
-						height: '50px',
-						padding: '13px 14px',
-						textAlign: 'center',
-						position: 'relative',
-						zIndex: 1,
```

Comment:
this is needed as well for the header to show while showing the webviews (efp and the incoming new success screen)